api/api/validator.py

```diff
@@ -340,155 +340,155 @@ def check_component_configuration_pair(component: str, configuration: str) -> Wa
 def format_active_response_command(command):
     if not is_safe_path(command):
         return False
     return check_exp(command, _active_response_command)
 
 
-@draft4_format_checker.checks("query")
+@Draft4Validator.FORMAT_CHECKER.checks("query")
 def format_query(value):
     return check_exp(value, _query_param)
 
 
-@draft4_format_checker.checks("range")
+@Draft4Validator.FORMAT_CHECKER.checks("range")
 def format_range(value):
     return check_exp(value, _ranges)
 
 
-@draft4_format_checker.checks("search")
+@Draft4Validator.FORMAT_CHECKER.checks("search")
 def format_search(value):
     return check_exp(value, _search_param)
 
 
-@draft4_format_checker.checks("sort")
+@Draft4Validator.FORMAT_CHECKER.checks("sort")
 def format_sort(value):
     return check_exp(value, _sort_param)
 
 
-@draft4_format_checker.checks("timeframe")
+@Draft4Validator.FORMAT_CHECKER.checks("timeframe")
 def format_timeframe(value):
     return check_exp(value, _timeframe_type)
 
 
-@draft4_format_checker.checks("wazuh_key")
+@Draft4Validator.FORMAT_CHECKER.checks("wazuh_key")
 def format_wazuh_key(value):
     return check_exp(value, _wazuh_key)
 
 
-@draft4_format_checker.checks("wazuh_version")
+@Draft4Validator.FORMAT_CHECKER.checks("wazuh_version")
 def format_wazuh_version(value):
     return check_exp(value, _wazuh_version)
 
 
-@draft4_format_checker.checks("date")
+@Draft4Validator.FORMAT_CHECKER.checks("date")
 def format_date(value):
     return check_exp(value, _iso8601_date)
 
 
-@draft4_format_checker.checks("date-time")
+@Draft4Validator.FORMAT_CHECKER.checks("date-time")
 def format_datetime(value):
     return check_exp(value, _iso8601_date_time)
 
 
-@draft4_format_checker.checks("hash_or_empty")
+@Draft4Validator.FORMAT_CHECKER.checks("hash_or_empty")
 def format_hash_or_empty(value):
     return True if value == "" else format_hash(value)
 
 
-@draft4_format_checker.checks("names_or_empty")
+@Draft4Validator.FORMAT_CHECKER.checks("names_or_empty")
 def format_names_or_empty(value):
     return True if value == "" else format_names(value)
 
 
-@draft4_format_checker.checks("numbers_or_empty")
+@Draft4Validator.FORMAT_CHECKER.checks("numbers_or_empty")
 def format_numbers_or_empty(value):
     return True if value == "" else format_numbers(value)
 
 
-@draft4_format_checker.checks("date-time_or_empty")
+@Draft4Validator.FORMAT_CHECKER.checks("date-time_or_empty")
 def format_datetime_or_empty(value):
     return True if value == "" else format_datetime(value)
 
 
-@draft4_format_checker.checks("group_names")
+@Draft4Validator.FORMAT_CHECKER.checks("group_names")
 def format_group_names(value):
     return check_exp(value, _group_names)
 
 
-@draft4_format_checker.checks("group_names_or_all")
+@Draft4Validator.FORMAT_CHECKER.checks("group_names_or_all")
 def format_group_names_or_all(value):
-    return check_exp(value, _group_names_or_all)
+    return check_exp(value, _group_names_or_all)
```

Comment:
Add EOF line

Reply:
Fixed in 8e322dd450925748ac5864038edb1c5146488d11.